app/iothub_client_sample_mqtt.cpp

```diff
@@ -83,6 +132,8 @@ void iothubInit()
         LogInfo("Failed on IoTHubClient_LL_SetDeviceTwinCallback");
         return;
     }
+
+    IoTHubClient_LL_SetDeviceMethodCallback(iotHubClientHandle, deviceMethodCallback, NULL);
```

Comment:
Keep aligned with failure check with IoTHubClient_LL_SetDeviceTwinCallback and IoTHubClient_LL_SetMessageCallback